samples/smart/README.md

```diff
@@ -1,33 +1,60 @@
-# Azure SMART on FHIR ONC (g)(10) Sample
+# Azure ONC (g)(10) & SMART on FHIR Sample
```

Comment:
Please reword to - Sample for SMART on FHIR Implementation Guide (v1.0.0)
§ 170.215(a)(3): HL7® SMART Application Launch Framework Implementation Guide Release 1.0.0 (https://hl7.org/fhir/smart-app-launch/1.0.0/)